test/parallel/test-runner-watch-mode-complex.mjs

```diff
@@ -0,0 +1,100 @@
+// Flags: --expose-internals
```

Comment:
Is this used only for `util.createDeferredPromise`?

Reply:
yes, it only used for `util.createDeferredPromise`